clippy_lints/src/formatting.rs

```diff
@@ -218,6 +218,17 @@ fn check_else(cx: &EarlyContext<'_>, expr: &Expr) {
         if let Some(else_pos) = else_snippet.find("else");
         if else_snippet[else_pos..].contains('\n');
         then {
+            // Allow allman style braces `if .. \n { \n .. \n } \n else \n { \n .. \n }`
+            if_chain! {
+                if is_block(else_);
+                if else_snippet.starts_with('\n');
```

Comment:
To allow for trailing spaces.
```suggestion
                if else_snippet[..else_pos].contains('\n');
```